Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -73,18 +78,21 @@ class FCircle
 	float DiameterCircle;
 public:
 	FCircle()
+	{
+	}
+	FCircle(float RadiusCircle,float DiameterCircle)
 	{
 		RadiusCircle = 0.0f;
 		DiameterCircle = 0.0f;
 	}
 	
-	void InPutDataCircle()
+	void InputDataCircle()
 	{
 		float EntryRadius;
 		float EntryDiameter;
 		std::cout << "what is the area of Your Circle:" << std::endl;
```

Comment:
I just noticed, you're asking for the area, not the radius.

Reply:
haha, sorry, but updated